001-IntroToKubernetes/Coach/03-k8sintro.md

```diff
@@ -10,21 +10,23 @@
 	- The default Kubernetes version used by the az aks create command should be fine.  
 	- The CLI should be used to create the cluster to give the most realistic experience.  
 	- Cluster names should be unique within the subscription.  
-	- Here’s an example command that creates a cluster named **wth-aks02-poc** in resource group **wth-rg02-poc:** using basic networking, managed identity, 3 nodes in separate availability zones and an attached ACR:
+	- Here’s an example command that creates a cluster named **wth-aks02-poc** in resource group **wth-rg02-poc:** using basic networking, managed identity, 3 nodes in separate availability zones and an attached ACR and doesn't generate any ssh keys:
 		- `az aks create --location eastus --name wth-aks02-poc --node-count 3  --no-ssh-key --resource-group wth-rg02-poc --zones 1 2 3 --enable-managed-identity --attach-acr <acrname>`
-    - Documentation on installing AKS can be found here:
+  	- **NOTE:** Attaching an ACR requires the student to have Owner or Azure account administrator role on the Azure subscription. If this is not possible then someone who is an Owner can do the attach for the student after they create the cluster.
+		- See below for the `az aks update` command that is used to attach the ACR.
+	- Documentation on installing AKS can be found here:
 		- [Portal](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough-portal)
 		- [CLI](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough)
 - It is usually a good idea to explain to the students what kind of options they have when creating a cluster. Doing a walkthrough demo of provisioning a cluster with the Portal is good showcasing tool, but end by telling them they need to figure out how to achieve the same thing with the CLI.
 - Have the teams show you the running cluster with:
 	- `kubectl get nodes`
-		- It should 3 nodes with each node in a different availability zone.  
+		- This show show three nodes, but it will not show the availability zone.  
+    - To see the availability zone, run:  `kubectl get nodes -o custom-columns=NAME:'{.metadata.name}',REGION:'{.metadata.labels.topology\.kubernetes\.io/region}',ZONE:'{metadata.labels.topology\.kubernetes\.io/zone}'`
```

Comment:
This is a spoonfeed IMO.  I think stick it in the coach's guide.

Personally, my go to here is to use 'kubectl describe node <nodename>' and let the students explore what data comes back.  They can google the fancy command that parses out just the zone #.  NO ONE will ever remember that as a takeaway from the hack! 

If you feel strongly about putting a pointer here, then go with 'describe node' and put a comment asking the student to review/explore the data that comes back.

Reply:
I agree with Peter... I don't see a reason to spoon feed them off the jump. Give it as a nugget to the coach to use if the students get stuck.

Reply:
Peter & Gino:  Maybe our changes are out of sync; The updates you're noting are only in the coach guide.